game.js

```diff
@@ -17,6 +17,7 @@ class Game {
         this.state = null;
         this.opponent_evenodd = null;
         this.greeted = false;
+        this.greetedbotcommand = false;
```

Comment:
i think these can be merged into one

Reply:
as long as we do our greeting only once yes
but if someday we want to move this botcommand greeting at another place, then we'd need a specific option

for now your advice seems the most fit to this case :100:

Reply:
edit: it makes the code a bit less clear, should we remove this, what does everyone think?

Reply:
Yes, merge this into one, and move `!this.greeted` in the check on line 371 to the check on line 370.

Reply:
tested to work
but do not merge yet, i need to commit last changes

Reply:
rebased and tested, if travis is good we can merge this